slidez-fe-core/src/pages/update/UpdatePage.tsx

```diff
@@ -41,14 +33,50 @@ export const UpdatePage: FC = () => {
         setIsActive(!isActive)
     }
 
-    const [userData, setUserData] = useState(userValues)
-    const [userPassword, setUserPassword] = useState(pwdValues)
+    const initialValues: User = {
+        id: '',
+        firstName: '',
+        lastName: '',
+        email: '',
+        password: '',
+    }
 
-    const handleInfoSubmit = (values: typeof userValues) => {
-        console.log(values)
+    const [userData, setUserData] = useState<User>(initialValues)
+
+    const schema = yup.object({
+        email: yup.string().email().max(64),
+        firstName: yup.string().required().min(3).max(30),
+        lastName: yup.string().required().min(3).max(30),
+    })
+    const currentToken =
+        'eyJhbGciOiJIUzUxMiJ9.eyJzdWIiOiJlbWFpbF8xMTJAdWtyLm5ldCIsImV4cCI6MTYyODg2NzY5Mn0.Y488vLEvUZrFV2mCv4V-W95z7SocWPgJgTs1xFZjXSNlW96jXYumdclEpzOPaXPd_ZwbkkBdUFBd9tFfeOOmBQ'
+
+    useEffect(() => {
+        axios
+            .get(
+                `http://localhost:5000/api/v1/users/userInfo?token=${currentToken}`
+            )
+            .then((response) => {
+                setUserData(response.data)
+                console.log(response.data)
+                console.log(userData)
+            })
+    }, [])
+
+    const handleInfoSubmit = (values: typeof userData) => {
+        axios
```

Comment:
No axios here - use http-helper

Reply:
Actually, write user service and user http helper there. In component you must dispatch action like "updateUser(dto)", and then in the async thunk call method of service. After user is updated (or not), process response in reducer

Reply:
This staff again